.changeset/five-birds-thank.md

```diff
@@ -0,0 +1,2 @@
+---
```

Comment:
What does this file do?

Reply:
This is a [changeset](https://github.com/changesets/changesets) file, it's a new bit of tooling that we've adopted in the past couple of months that helps automatically generate changelog entries and version bumps.